editor/src/components/canvas/controls/elements-outside-visible.spec.browser2.tsx

```diff
@@ -106,7 +106,7 @@ describe('elements outside visible area', () => {
       const indicator = screen.queryByTestId(getIndicatorId(targetPath, ['right']))
       expect(indicator).not.toBeNull()
       expect(indicator?.style.top).toBe('182px')
-      expect(indicator?.style.left).toBe('1920px')
+      expect(indicator?.style.left).toBe('2175px')
```

Comment:
Bah, it looks like this PR breaks the arrow indicator when the selected element is off-screen to the right